### PR TITLE
docs: add readme, changelog, up to version gradle

### PR DESCRIPTION
**Pull request by gyuguen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 TBD
 
+## [v2.0.1](https://github.com/medibloc/panacea-java/releases/tag/v2.0.1) - 2021-08-11
+
+### Bug fixs
+- -[\#47] feat: Add getTx() and getTxs() for fetching a memo and so on
```
Comment (youngjoon-lee) [inline CHANGELOG.md]: Please add a `fix: Include a memo...` PR as well. Also, put this `feat: ...` message to the `Features`.

**Review by youngjoon-lee — COMMENTED** [review]
Please set the commit message as `Release v2.0.1`, so that we can track the history easily in the future.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,11 @@
 
 TBD
 
+## [v2.0.1](https://github.com/medibloc/panacea-java/releases/tag/v2.0.1) - 2021-08-11
+
+### Bug fixs
```
Comment (youngjoon-lee) [inline CHANGELOG.md]: Typo. `Bug fixes`

File: CHANGELOG.md (hunk 1 above)
Comment (gyuguen) [reply CHANGELOG.md]: Yes, I modified it.

Comment (gyuguen) [reply CHANGELOG.md]: I modified it again because it was incorrect.

File: CHANGELOG.md (hunk 2 above)
Comment (gyuguen) [reply CHANGELOG.md]: OK. I modified it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,15 @@
 
 TBD
 
+## [v2.0.1](https://github.com/medibloc/panacea-java/releases/tag/v2.0.1) - 2021-08-11
+
+### Features
+- [\#47] feat: Add getTx() and getTxs() for fetching a memo and so on
+- [\#48] feat: Add check memo when test multi sends. and fee check.
```
Comment (gyuguen) [inline CHANGELOG.md]: Delete this feature because it does not seem necessary to specify it.

**Review by youngjoon-lee — APPROVED** [review]
ACK

**Pull request merged**